diff --git a/ios/App/App/capacitor.config.json b/ios/App/App/capacitor.config.json
index 10b434f..124b7e3 100644
--- a/ios/App/App/capacitor.config.json
+++ b/ios/App/App/capacitor.config.json
@@ -6,20 +6,31 @@
 		"url": "https://www.celimap.com.ar",
 		"cleartext": false
 	},
+	"android": {
+		"appendUserAgent": " CelimapNative/1",
+		"backgroundColor": "#0b1220",
+		"allowMixedContent": false
+	},
+	"ios": {
+		"appendUserAgent": " CelimapNative/1",
+		"backgroundColor": "#0b1220"
+	},
 	"plugins": {
 		"SplashScreen": {
-			"launchShowDuration": 1500,
+			"launchShowDuration": 0,
 			"launchAutoHide": true,
 			"backgroundColor": "#0b1220",
 			"showSpinner": false
 		},
 		"StatusBar": {
 			"style": "DARK",
-			"backgroundColor": "#0b1220"
+			"backgroundColor": "#0b1220",
+			"overlaysWebView": true
 		}
 	},
 	"packageClassList": [
 		"AppPlugin",
+		"CAPBrowserPlugin",
 		"SplashScreenPlugin",
 		"StatusBarPlugin"
 	]
diff --git a/ios/App/CapApp-SPM/Package.swift b/ios/App/CapApp-SPM/Package.swift
index 2df3dd5..bf33a14 100644
--- a/ios/App/CapApp-SPM/Package.swift
+++ b/ios/App/CapApp-SPM/Package.swift
@@ -13,6 +13,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", exact: "8.4.2"),
         .package(name: "CapacitorApp", path: "..\..\..\node_modules\@capacitor\app"),
+        .package(name: "CapacitorBrowser", path: "..\..\..\node_modules\@capacitor\browser"),
         .package(name: "CapacitorSplashScreen", path: "..\..\..\node_modules\@capacitor\splash-screen"),
         .package(name: "CapacitorStatusBar", path: "..\..\..\node_modules\@capacitor\status-bar")
     ],
@@ -23,6 +24,7 @@ let package = Package(
                 .product(name: "Capacitor", package: "capacitor-swift-pm"),
                 .product(name: "Cordova", package: "capacitor-swift-pm"),
                 .product(name: "CapacitorApp", package: "CapacitorApp"),
+                .product(name: "CapacitorBrowser", package: "CapacitorBrowser"),
                 .product(name: "CapacitorSplashScreen", package: "CapacitorSplashScreen"),
                 .product(name: "CapacitorStatusBar", package: "CapacitorStatusBar")
             ]
